mass_merge/models/ir_ui_view.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# © 2020 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import models
+
+
+class View(models.Model):
+
+    _inherit = 'ir.ui.view'
+
+    def postprocess_deactivated(
```

Comment:
The correct method should be `postprocess`
[See here](https://github.com/OCA/OCB/blob/14.0/odoo/addons/base/models/ir_ui_view.py#L925)

Please take note of the arguments as well.

Reply:
Working on it

Reply:
I changed the method into postprocess